src/ActionModal/utils/ActionManager.js

```diff
@@ -12,17 +12,14 @@ const txFetchOptions = {
   }
 }
 
-const apiURL = config.graphqlHost
-console.log(config, apiURL)
-
 async function transactionAPIRequest(payload) {
   // console.log(config)
   const options = {
     ...txFetchOptions,
     body: JSON.stringify({ payload })
   }
 
-  return fetch(`${config.graphqlHost}/transaction`, options).then(r => r.json())
+  return fetch(`${config.graphqlHost}/transaction/estimate`, options).then(r => r.json())
```

Comment:
we have axios as a dependency. would you rather prefer using fetch?

Reply:
I would personally. It's supported in all modern browsers.

The reason I used it here was just convenience though to get something running.